spec/controllers/application_controller_spec.rb

```diff
@@ -53,6 +73,41 @@ def append_info_to_payload(payload)
 
   it_behaves_like 'a sentry logger'
 
+  describe 'Sentry Handling' do
+    around do |example|
+      with_settings(Settings.sentry, dsn: 'T') do
+        example.run
+      end
+    end
+
+    it 'does not log to sentry if Common::Exceptions::Unauthorized' do
+      expect(Raven).not_to receive(:capture_exception)
+      expect(Raven).not_to receive(:capture_message)
+      get :unauthorized
+    end
+
+    it 'does not log to sentry if Common::Exceptions::RoutingError' do
+      expect(Raven).not_to receive(:capture_exception)
+      expect(Raven).not_to receive(:capture_message)
+      get :routing_error
```

Comment:
if it were me I would want this logged as "info" or "warning" -- I don't know why we want to exclude it from Sentry altogether.